lib/specs/latest.js

```diff
@@ -1,25 +1,29 @@
 const _ = require('lodash');
 const previousSpec = require('./v1');
+const ghostVersions = require('../utils').versions;
+const latestDocsBaseUrl = `https://themes.ghost.org/docs/`;
+const prevDocsBaseUrl = `https://themes.ghost.org/v${ghostVersions.v1.docs}/docs/`;
+const prevDocsBaseUrlRegEx = new RegExp(prevDocsBaseUrl);
 
 const previousKnownHelpers = previousSpec.knownHelpers;
 const previousTemplates = previousSpec.templates;
 const previousRules = previousSpec.rules;
 
-let knownHelpers;
-let templates;
-let rules;
-
-knownHelpers = [];
+let knownHelpers = [];
+let templates = [];
+let rules = {};
 
 knownHelpers = _.union(previousKnownHelpers, knownHelpers);
 
-templates = [];
-
 templates = _.union(previousTemplates, templates);
 
-rules = {};
-
-rules = _.merge(previousRules, rules);
+// Merge the previous rules into the new rules, but overwrite any specified property,
+// as well as adding any new rule to the spec.
+// Furthermore, replace the usage of the old doc URLs that we're linking to, with the
+// new version.
+rules = _.each(_.merge(previousRules, rules), function replaceDocsUrl(value) {
+    value.details = value.details.replace(prevDocsBaseUrlRegEx, latestDocsBaseUrl);
```

Comment:
There is a "g" missing the regex. I can still see some 1.25 links when i scan the latest version.

Reply:
Thanks! Added and pushed. It should be correct now.